templates/test-sw.hbs

```diff
@@ -44,22 +44,22 @@
 
     <script>
       mocha.setup({
-        ui: 'bdd'
+        ui: 'bdd',
+        timeout: 10000,
```

Comment:
I set this because when debugging it was annoying to get timeout errors. Then I noticed a 10 second timeout was in the browser template, so I kept it for consistency.

I can revert if you like.